src/adapter/mod.rs

```diff
@@ -220,7 +223,7 @@ impl<'a> Adapter<'a> for RustdocAdapter<'a> {
         let coerce_to_type = coerce_to_type.clone();
         match type_name.as_ref() {
             "Item" | "Variant" | "FunctionLike" | "Importable" | "ImplOwner" | "RawType"
-            | "ResolvedPathType" | "GlobalValue" => {
+            | "ResolvedPathType" | "GlobalValue" | "AssociatedType" => {
```

Comment:
I think this isn't necessary, `AssociatedType` isn't an interface so it shouldn't ever be in a situation where we're coercing _from_ it.
```suggestion
            | "ResolvedPathType" | "GlobalValue" => {
```

Reply:
fixed in 2329a50